### DISCUSSION TITLE
Error: No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()

### DISCUSSION THREAD

**Discussion by zoobibackups** [invertase/react-native-firebase#5992] [Q&A] [answered]
Google signIn working correctly but **auth().signInWithCredential(googleCredential)** causes this error.
```
try {
      await GoogleSignin.hasPlayServices();
      const userInfo = await GoogleSignin.signIn(); // working
      const googleCredential = auth.GoogleAuthProvider.credential(userInfo?.idToken); // working
      auth().signInWithCredential(googleCredential).then((data) => {     // this generate error
           console.log(data)
      })
  } catch (error) {
      window.alert(error) 
  }
```
The error log: 

**`Error: No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()`**

I don't know what i am missing here. Do I need any configuration for it??
Thanks in advance!

**Comment by andersonaddo** [comment, marked as answer]
Duplicate of https://github.com/invertase/react-native-firebase/issues/5993.

**Reply by zoobibackups** [reply]
No solution provided yet

**Reply by zoobibackups** [reply]
The duplicate issue is closed without resolving

**Reply by mikehardy** [reply]
@zoobibackups you persist in commenting and thumbs-downing etc without following *any* of our advice on how to successfully access help and how to successfully help yourself.

This comment is authoritative, and if you follow the advice it will help you.

https://github.com/invertase/react-native-firebase/issues/5993#issuecomment-1006529749

To be crystal clear: no one will provide you with a solution. You will provide yourself with a solution, based on our advice. We have given you the advice you need. You now need to get to work and find the solution yourself, using the advice.
